### ISSUE TITLE
Missing margin-bottom on large lists with responsive layouts. 

### ISSUE THREAD

**Issue by bwh-rl** [PolymerElements/iron-list#280]
### Description
In an responsive layout with % item width and fixed item aspect ratios heights will have subpixel.
The _rowHeight is measured with offsetHeight, which does not consider subpixel.
This leads to an wrong _estScrollHeight, and the missing of the margin-bottom.

### Live Demo
https://jsbin.com/zifijenavu/edit?html,output

### Steps to reproduce
Scroll to the bottom

### Browsers Affected
Chrome
not tested, but should affect all browsers.

**Comment by blasten** [comment]
yes, it could use `item.getBoundingClientRect().height` in https://github.com/PolymerElements/iron-list/blob/master/iron-list.html#L1241  similar to L1239.

**Comment by keanulee** [comment]
May not be a clear fix - may have perf implications
